docs/rules/max-depth.md

```diff
@@ -1,68 +1,70 @@
-# Limit Maximum Depth (max-depth)
+# enforce a maximum depth that blocks can be nested (max-depth)
 
 The `max-depth` rule allows you to specify the maximum depth blocks can be nested.
```

Comment:
Since the existing introduction sentence is redundant with the description in the main heading and Rule Details, how about something adapted from max-nested-callbacks?

Many developers consider code difficult to read if blocks are nested beyond a certain depth.

This introduction example is also redundant, so how about delete it, but put that style of comment in the **incorrect** and **correct** code.